packages/fxa-auth-server/test/local/senders/mjml-emails.ts

```diff
@@ -163,9 +163,7 @@ const TESTS: [string, any][] = [
   
   ['lowRecoveryCodesEmail', new Map<string, Test | any>([
     ['subject', [
-      { test: 'include', expected: '2' },
-      { test: 'include', expected: 'recovery codes remaining' },
-      { test: 'notInclude', expected: '1' },
+      { test: 'include', expected: '2 recovery codes remaining' }
```

Comment:
Since we disabled the `useIsolating` option in fluent, we no longer need to check for specific values and can rather test for the entire phrase together just like the old tests